packages/cpt-ui/src/components/prescriptionDetails/ItemsCards/SummaryListRow.tsx

```diff
@@ -1,8 +1,9 @@
 import {SummaryList} from "nhsuk-react-components"
+import React from "react"
 
 interface SummaryListRowProps {
   readonly label: string
-  readonly value: string
+  readonly value: React.ReactNode
 }
 
 export const SummaryListRow = ({label, value}: SummaryListRowProps) => {
```

Comment:
Could we update the Summary List Row component to take an optional tag, and then use the NHSUK tag component something like?:

```suggestion
import {TagColour} from "@/helpers/statusMetadata"
import {SummaryList, Tag} from "nhsuk-react-components"
interface SummaryListRowProps {
  readonly label: string
  readonly value?: string
  readonly tagValue?: string
  readonly tagColour?: TagColour
}

export const SummaryListRow = ({label, value, tagValue, tagColour}: SummaryListRowProps) => {

  return (
    <SummaryList.Row>
      <SummaryList.Key>{label}</SummaryList.Key>
      <SummaryList.Value>
        <span className="data-field__content data-field__content--address">
          {value && (value)}
          {tagValue && (
          <Tag color={tagColour}>
            {tagValue}
          </Tag>
        )}
        </span>
      </SummaryList.Value>
    </SummaryList.Row>
  )
}
```